backend/package.json

```diff
@@ -11,7 +11,7 @@
     "test": "jest"
   },
   "dependencies": {
-    "@autonomys/auto-drive": "^0.5.3",
+    "@autonomys/auto-drive": "^0.6.2",
```

Comment:
Updated because there was an issue in the former version generating file chunk list: https://github.com/autonomys/auto-sdk/pull/112